src/playground/sample_data_ingestion/sample_data_indices/sample_data_indices.py

```diff
@@ -0,0 +1,34 @@
+from utils.sample_data_ingestor.sample_data_ingestor import ingest
+
+"""
+Index class: this class can create, delete, and ingest more data into a new or existing index
+Arguments:
+    - index_body: The body containing the configurations of the index (used for index creation)
+    - index_name: The index name to create/ingest/delete
+    - client: an OpenSearch Python client object
+"""
+class SampleDataIndex:
+    def __init__(self, index_name, client, index_body):
+        self.index_name = index_name
+        self.index_body = index_body
+        self.client = client
```

Comment:
if possible, please add some logic to validate it. Thanks

Reply:
Done, type checking is enforced for the variables